src/agent/src/uevent.rs

```diff
@@ -240,7 +240,6 @@ pub(crate) fn spawn_test_watcher(sandbox: Arc<Mutex<Sandbox>>, uev: Uevent) {
                     if matcher.is_match(&uev) {
                         let (_, sender) = watch.take().unwrap();
                         let _ = sender.send(uev.clone());
-                        return;
```

Comment:
This clippy fix should be on a separate commit really.

Reply:
With seperated commit now :)